### PR TITLE
CI clean up

### PR DESCRIPTION
**Pull request by szabinst**
refs: MBL-16546
affects: Student, Teacher, Parent
release note: Testing
test plan: none

## Checklist

- [ ] Follow-up e2e test ticket created or not needed
- [ ] A11y checked
- [ ] Tested on phone
- [ ] Tested on tablet
- [ ] Tested in dark mode
- [ ] Tested in light mode
- [ ] Approve from product or not needed

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Parent Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Teacher Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<details><summary>Student Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.92% | 91.91% | 0.01%
Core/Core/Courses/CourseDetails/ViewModel/CourseDetailsViewModel.swift | 48.89% | 48.89% | 0%

[MBL-16546](https://instructure.atlassian.net/browse/MBL-16546)

#### Affected Apps: Student, Teacher, Parent

#### Release Note: 
Testing
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by vargaat — CHANGES_REQUESTED** [review]

File: TestsFoundation/TestsFoundation/CoreUIElements/SpringBoard/SpringBoard.swift [hunk 1]
```diff
@@ -68,26 +55,23 @@ public class SpringBoard {
...
         button.tap()
     }
 
     public func setupSplitScreenWithSafariOnRight() {
         resetMultitasking()
 
-        bringUpDock()
+        var splitViewButtonID = ""
+        if Bundle.main.isTeacherTestsRunner {
+            splitViewButtonID = "top-affordance:com.instructure.ios.teacher"
```
[the remark this answers is not in the document]
Comment (szabinst) [reply TestsFoundation/TestsFoundation/CoreUIElements/SpringBoard/SpringBoard.swift]: fixed!

File: TestsFoundation/TestsFoundation/CoreUIElements/SpringBoard/SpringBoard.swift [hunk 2]
```diff
@@ -68,26 +55,23 @@ public class SpringBoard {
...
+        } else if Bundle.main.isStudentTestsRunner {
+            splitViewButtonID = "top-affordance:com.instructure.icanvas"
```
[the remark this answers is not in the document]
Comment (szabinst) [reply TestsFoundation/TestsFoundation/CoreUIElements/SpringBoard/SpringBoard.swift]: fixed!

**Review by idevainstr — APPROVED** [review]

**Pull request merged**